lib/TPP/ToBlockLayoutAndBack.cpp

```diff
@@ -85,6 +85,17 @@ static Value handleLayout_VNNI(OpBuilder &builder, Location loc, Value input,
                             /*outerDimsPerm=*/{});
 }
 
+static Value handleBRGemmLayout_VNNI(OpBuilder &builder, Location loc,
+                                     Value input, Value output,
+                                     ArrayRef<OpFoldResult> tiles) {
+  assert(tiles.size() == 1 && "expect 1 block for VNNI");
+  SmallVector<int64_t> innerDimPos = {1};
+  if (!output)
```

Comment:
IIRC, this distinction is about tensor vs memref semantics right? If the requirement below (need to be on tensors) is right, than we really don't need this distinction.

Reply:
I have removed the unpack call from here, it doesn't make sense indeed.